medical/models/medical_patient.py

```diff
@@ -19,39 +19,13 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 # #############################################################################
-
-from openerp.osv import fields, orm
-
-from dateutil.relativedelta import relativedelta
+from openerp import models, fields, api
 from datetime import datetime
 from openerp.tools.translate import _
+from dateutil.relativedelta import relativedelta
 
 
-class MedicalPatientMedCenterRel(orm.Model):
-    _name = 'medical.patient.med.center.rel'
-    _rec_name = 'patient_id'
-    _columns = {
-        'patient_id': fields.many2one('medical.patient', 'Patient',
-                                      required=True,
-                                      select=1, ondelete='cascade'),
-        'medical_center_id': fields.many2one('res.partner', 'Medical Center',
-                                             required=True, select=1,
-                                             ondelete='cascade',
-                                             domain="[('is_institution',"
-                                                    "  '=', True)]"),
-        'identification_code': fields.char(size=256, string='ID',
-                                           help='Patient Identifier '
-                                                'provided by the '
-                                                'Health Center.Is not the '
-                                                'Social Security Number'),
-    }
-    _sql_constraints = [('uniq_patient_center',
-                         'unique(patient_id, medical_center_id)',
-                         "You cannot have twice a medical "
-                         "center for a patient"), ]
-
-
-class MedicalPatient(orm.Model):
+class MedicalPatient(models.Model):
     '''
     The concept of Patient included in medical.
 
```

Comment:
You should replace in the next line in the class description, 'User' with 'Partner'
```
A patient is an User with extra elements 
```